includes/wikia/models/tests/DesignSystemGlobalFooterModelTest.php

```diff
@@ -30,6 +30,7 @@ public function testGetLicensingAndVertical( $sitename, $rightsText, $rightsUrl,
 	public function getLicensingAndVerticalDataProvider() {
 		return [
 			[
+				DesignSystemGlobalFooterModel::PRODUCT_WIKIS,
 				'wikia',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wikia" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)